mpi-proxy-split/mpi-wrappers/generate-mpi-fortran-wrappers.py

```diff
@@ -90,13 +90,17 @@ def emit_wrapper(decl, ret_type, fnc, args, arg_vars):
   if fnc in ['MPI_Wtime', 'MPI_Wtick']:
     print("  return " + fnc + "(" + cargs + ");")
   else:
+      if "int* index" in fargs:
```

Comment:
Is this reliable? Does this always mean the "index" of an array?

Reply:
From what we've seen so far, yes - only `MPI_Testany` and `MPI_Waitany` use `int* index` as an argument in the wrappers we have implemented.